daemon/policy_test.go

```diff
@@ -272,10 +272,10 @@ func (ds *DaemonSuite) TestUpdateConsumerMap(c *C) {
 						RemotePolicies: expectedRemotePolicies,
 						L7:             &PNPAllowAll,
 					},
-					{
-						RemotePolicies: expectedRemotePolicies,
-						L7:             &PNPAllowGETbar,
-					},
+					//{
```

Comment:
I changed the l4 rule processing to always include any selectors in the l7rulesPerEp also in CachedSelectors, and never include duplicates in CachedSelectors, and figured that this would be OK as it is shadowed by the rule above.
I think this change is due to not having duplicate selectors in the former `Endpoints`. Like 85% sure :-)

Reply:
Yeah, as long as we are redirecting to the proxy, we should be OK, since the shadowing is done at L7 itself.